modules/login/src/main/resources/META-INF/resources/login/js/login.js

```diff
@@ -0,0 +1,64 @@
+var loginpage = function($, w, moment) {
+
+    var pageData = {};
+
+    function setPageDate(data) {
+        pageData = data;
+    }
+    function switchWorkspace(selectobj) {
+        // document.getElementById("currentWorkspace").value = newWorkspace;
+        if ($(selectobj.target).val() != " ") {
+            $('#currentWorkspace').val($(selectobj.target).val());
+            document.forms["frmWorkspace"].submit();
+        }
+    }
+
+    function setWorkspace() {
+        //set the list of workspaces.
```

Comment:
Comments like this can indicate a function that should be extracted for readability. Consider extracting a function here.